lib/src/HttpResponseImpl.cc

```diff
@@ -209,7 +209,8 @@ HttpResponsePtr HttpResponse::newFileResponse(
     const unsigned char *pBuffer,
     size_t bufferLength,
     const std::string &attachmentFileName,
-    ContentType type)
+    ContentType type,
+    const std::string &typeString)
```

Comment:
should we get the typeString in the newFileResponse method internally? if user want use a customed string, he can use the registerCustomExtensionMime method.

Reply:
No. The static file router does not pass the file name to newFileResponse. Thus we cannot infer the content type when creating the response. 

We can make StaticFileRouter set the content type after response creation. But I think that's dirty.

Reply:
but now the `parseContentType` method is called twice (in the StaticFileRouter and in the fileNameToMime), 
It's a bit inelegant.

Reply:
`fileNameToMime` should have called getConetntType. That's a bug.